tests/includes/StripeTerminalServiceTest.php

```
@@ -238,17 +238,113 @@ public function test_set_stripe_client_replaces_existing_client(): void {
 	/**
 	 * Test that set_stripe_client returns void.
 	 */
-	public function test_set_stripe_client_returns_void(): void {
-		$service = new StripeTerminalService( 'sk_test_fake_key_123' );
-		$client  = new \Stripe\StripeClient( 'sk_test_key' );
+	public function test_set_stripe_client_returns_void(): void {
+		$service = new StripeTerminalService( 'sk_test_fake_key_123' );
+		$client  = new \Stripe\StripeClient( 'sk_test_key' );
 
 		$result = $service->set_stripe_client( $client );
```

PHPStan (2.2.7)

[error] Result of method WCPOS\WooCommercePOS\StripeTerminal\StripeTerminalService::set_stripe_client() (void) is used.


(method.void)